homeassistant/components/goalzero/const.py

```diff
@@ -25,6 +48,26 @@
     "inputDetected": ["Input Detected", DEVICE_CLASS_POWER, None],
 }
 
+SENSOR_DICT = {
+    "wattsIn": ["Watts In", DEVICE_CLASS_POWER, POWER_WATT, True],
+    "ampsIn": ["Amps In", DEVICE_CLASS_CURRENT, ELECTRICAL_CURRENT_AMPERE, False],
+    "wattsOut": ["Watts Out", DEVICE_CLASS_POWER, POWER_WATT, True],
+    "ampsOut": ["Amps Out", DEVICE_CLASS_CURRENT, ELECTRICAL_CURRENT_AMPERE, False],
+    "whOut": ["WH Out", DEVICE_CLASS_ENERGY, ENERGY_WATT_HOUR, False],
+    "whStored": ["WH Stored", DEVICE_CLASS_ENERGY, ENERGY_WATT_HOUR, True],
+    "volts": ["Volts", DEVICE_CLASS_VOLTAGE, VOLT, False],
+    "socPercent": ["State of Charge Percent", DEVICE_CLASS_BATTERY, PERCENTAGE, True],
+    "timeToEmptyFull": ["Time to Empty/Full", TIME_MINUTES, TIME_MINUTES, True],
+    "temperature": ["Temperature", DEVICE_CLASS_TEMPERATURE, TEMP_CELSIUS, True],
+    "wifiStrength": [
+        "Wifi Strength",
+        DEVICE_CLASS_SIGNAL_STRENGTH,
+        SIGNAL_STRENGTH_DECIBELS,
+        True,
+    ],
+    "timestamp": ["Up Time", TIME_SECONDS, TIME_SECONDS, False],
```

Comment:
I don't think that `TIME_SECONDS` can be used as device class, perhaps just put `None` there?